addons/editor-number-arrow-keys/addon.json

```
@@ -3,6 +3,12 @@
   "description": "Use the up and down arrow keys to increase or decrease number values in block inputs.",
   "tags": ["editor", "codeEditor", "recommended"],
   "versionAdded": "1.34.0",
+  "latestUpdate": {
+    "version": "1.35",
+    "isMajor": true,
+    "newSettings": ["useCustom", "regularCustom", "shiftCustom", "altCustom"],
```

I mean, technically all four are new, but...

```suggestion
    "newSettings": ["useCustom"],
```